packages/engine/lib/flatbuffers_gen/Cargo.toml

```
@@ -0,0 +1,7 @@
+[package]
+name = "flatbuffers_gen"
+description = "Flatbuffer binding for rust generated by flatc"
```

Not sure they're just bindings. Maybe just something like
```suggestion
description = "Generated flatbuffers code for specification of communication with Python."
```